recipes/corrade/all/test_package/conanfile.py

```diff
@@ -22,3 +22,7 @@ def test(self):
             if self.options["corrade"].with_utility:
                 # Run corrade-rc
                 self.run("corrade-rc --help", run_environment=True)
+
+        if self.settings.os == "Emscripten":
+            bin_path = os.path.join("bin", "test_package.js")
+            self.run("node {}".format(bin_path), run_environment=True)
```

Comment:
do we need https://conan.io/center/nodejs here?

Reply:
No, it is already a requirement when using Emscripten.